__tests__/app.test.js

```diff
@@ -73,6 +73,12 @@ describe('/api',()=>{
                 expect(body.msg).toBe('Bad Request')
             })
         })
+        test('GET:200 /articles/1 Should respond with the comment_count property',()=>{
+            return request(app).get('/api/articles/1').expect(200).then(({body})=>{
+                expect(body.article).toHaveProperty('comment_count');
```

Comment:
I would pop the other properties here too and also expect comment_count to be a number/string depending on how you did it :)